HISTORY.md

```diff
@@ -0,0 +1,199 @@
+<!--
+SPDX-FileCopyrightText: 2020 Helmholtz Centre for Environmental Research (UFZ)
+SPDX-FileCopyrightText: 2020 Helmholtz-Zentrum Dresden-Rossendorf (HZDR)
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+# Changelog
+
+## [Unreleased](https://gitlab.com/hifis/ansible/gitlab-role/tree/HEAD)
+
+[List of commits](https://gitlab.com/hifis/ansible/gitlab-role/-/compare/v1.1.0...main)
+
+### Changed
```

Comment:
```suggestion
**Changed**
```

Reply:
Dear Norman @Normo, thank you for your suggestion. I changed the HISTORY file and re-generated the CHANGELOG file. Is this fine from your point of view? Thanks for checking again.